vispy/visuals/markers.py

```diff
@@ -474,7 +474,6 @@
     '^': triangle_up,
     'v': triangle_down,
     '*': star,
-    None: None,
 }
 marker_types = tuple(sorted(list(_marker_dict.keys())))
 
```

Comment:
That's a good point.. this was added so that we could have `LinePlot(symbol=None)`, but perhaps it's better for the LinePlot to catch the `None` and hide the Markers?